services/src/unit-test/java/uk/gov/moj/sdt/services/RetryMessageAlreadySentServiceTest.java

```diff
@@ -232,158 +235,139 @@ public void queuePendingMessageExceededForwardingAttempts() {
+        verify(mockCacheable).getValue(IGlobalParameter.class, MAX_FORWARDING_ATTEMPTS);
+        verify(mockMessagingUtility, times(2)).enqueueRequest(any());
 
-        Assert.assertTrue("Test completed succesfully", true);
+        assertEquals(0, returnedListOfRequests.get(0)
+                .getForwardingAttempts(), FORWARDING_ATTEMPTS_ON_INDIVIDUAL_REQUEST);
+        assertEquals(IIndividualRequest.IndividualRequestStatus.RECEIVED
+                .getStatus(), returnedListOfRequests.get(0).getRequestStatus(), STATUS_SET_TO_RECEIVED);
 
+        assertTrue(true, TEST_COMPLETED_SUCCESSFULLY);
     }
 
     /**
      * This method tests the queue pending message method and the condition that
      * there are no pending messages to return from database.
      */
     @Test
-    public void queuePendingMessageWhenNoMessageFound() {
-        final List<IIndividualRequest> individualRequests = new ArrayList<IIndividualRequest>();
+    void queuePendingMessageWhenNoMessageFound() {
+        final List<IIndividualRequest> individualRequests = new ArrayList<>();
 
         final IGlobalParameter maxForwardingAttemptsParam = new GlobalParameter();
-        maxForwardingAttemptsParam.setName("MAX_FORWARDING_ATTEMPTS");
+        maxForwardingAttemptsParam.setName(MAX_FORWARDING_ATTEMPTS);
         maxForwardingAttemptsParam.setValue("3");
-        EasyMock.expect(this.mockCacheable.getValue(IGlobalParameter.class, "MAX_FORWARDING_ATTEMPTS")).andReturn(
+        when(this.mockCacheable.getValue(IGlobalParameter.class, MAX_FORWARDING_ATTEMPTS)).thenReturn(
                 maxForwardingAttemptsParam);
 
-        final int maxForwardingAttempts = Integer.valueOf(maxForwardingAttemptsParam.getValue());
+        final int maxForwardingAttempts = Integer.parseInt(maxForwardingAttemptsParam.getValue());
 
-        EasyMock.expect(mockIndividualRequestDao.getPendingIndividualRequests(maxForwardingAttempts)).andReturn(
+        when(mockIndividualRequestDao.getPendingIndividualRequests(maxForwardingAttempts)).thenReturn(
                 individualRequests);
 
-        EasyMock.replay(mockIndividualRequestDao);
-
         this.messageTaskService.queueMessages();
 
-        EasyMock.verify(mockIndividualRequestDao);
-
-        Assert.assertTrue("Method test completed successfully", true);
+        verify(mockIndividualRequestDao).getPendingIndividualRequests(maxForwardingAttempts);
 
+        assertTrue(true, "Method test completed successfully");
     }
 
     /**
      * This method tests the all successful scenario of the queue pending message
      * when the max forwarding attempts global parameter is not defined in the database.
      */
     @Test
-    public void queuePendingMessageForwardingAttemptsNotAvailable() {
-        final String sdtRequestRef = "TEST_1";
+    void queuePendingMessageForwardingAttemptsNotAvailable() {
+        final String sdtRequestRef = TEST_1;

```

Comment:
As a constant has been created for the SDT request value this variable is redundant. It can be removed and the constant used in it's place.

Reply:
replaced

Reply:
Looks like variable hasn't been removed from this method.

Reply:
removed now

Reply:
Confirmed variable has been removed and replaced with constant.